tools/mksurfdata_map/src/mksurfdat.F90

```diff
@@ -802,6 +806,9 @@ program mksurfdat
        landfrac_pft(n) = pctlnd_pft(n)/100._r8
     end do
 
+    pctnatpft_max = pctnatpft
```

Comment:
I'd find it more clear if these settings of pctnatpft_max and pctcft_max were done further down, within the conditional:

```fortran
    if (mksrf_fdynuse /= ' ') then
```